apps/ui/src/ui/form/EntityIdPicker.tsx

```diff
@@ -76,7 +76,6 @@ export function EntityIdPicker<T extends { id: number }>({
                 <span className={clsx('ui-text-field', size ?? 'regular')} style={{ flexGrow: 1 }}>
                     <Combobox.Input
                         displayValue={(value: T) => value && displayValue(value)}
-                        disabled={Boolean(value && !entity)}
```

Comment:
Apparently this disabled state does nothing according to the library owners. If its still needed would need to incorporate this logic into the higher up ComboBox params instead of just the passed in value